requirements.txt

```diff
@@ -0,0 +1,2 @@
+twilio==5.2.0
+phonenumbers==7.2.3
```

Comment:
Twilio is the SMS provider we're using, and ``phonenumbers`` is the Python port of the Google library for parsing and formatting phone numbers.  We need to make sure that all phone numbers are in ``E.164`` format before sending them to Twilio.